README.md

```diff
@@ -7,11 +7,8 @@ module with OpenAM see the chapter,
 *[Customizing Authentication Modules](http://openam.forgerock.org/doc/bootstrap/dev-guide/#sec-auth-spi)*,
 in the OpenAM *Developer's Guide*.
 
-Before building the sample authentication module, update
-the OpenAM version in the POM to match the version of OpenAM
-that you are using. The line to update is:
-
-    <openam.version>13.0.0-SNAPSHOT</openam.version>
+Note that this branch is to OpenAM 13.0.x custom module. Please checkout the right branches if you're not using this
```

Comment:
I think it would be clearer if you change the wording for the first sentence:

    this branch is for building a custom module with OpenAM 13.0.x.